scripts/ci/test_extensions.sh

```diff
@@ -24,7 +24,11 @@ exit_code=0
 # Disable k8s-extension temporarily: https://github.com/Azure/azure-cli-extensions/pull/6702
 # Disable alias temporarily: https://github.com/Azure/azure-cli/pull/27717
 # hybridaks is going to be deprecated: https://github.com/Azure/azure-cli/pull/29838
-ignore_list='azure-cli-ml fzf arcappliance arcdata connectedk8s k8s-extension alias hybridaks'
+# db-up is going to be deprecated: https://github.com/Azure/azure-cli/pull/29887
+ignore_list='azure-cli-ml fzf arcappliance arcdata connectedk8s k8s-extension alias hybridaks db-up'
```

Comment:
`db-up`'s `pymssql==2.2.7` does not provide wheel for 3.12. This extension is deprecated.
https://github.com/Azure/azure-cli-extensions/blob/c55f55aaf70fd5df20b5a789294bb909aab4d1ce/src/db-up/setup.py#L33